site/zenodo_rdm/utils.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Utility functions."""
+
+
+def github_link_render(record):
+    """Entry for Github."""
+    # [TODO] Integrate Inveio_Github here
+    resources = []
+
+    related_identifiers = record["ui"].get("related_identifiers", [])
```

Comment:
Little (maybe stupid) comment/quiestion: do we know for sure that `record["ui"]` exists?
Is it better to use `lodash/get` here instead?

Reply:
from tests and debugging it looks to me like `ui` always exists although this doesn't seem to be the case for `related_identifiers` (hence the `get`).
But you're right using `get` for both is probably safer